src/ontology/uberon-edit.obo

```diff
@@ -224421,7 +224423,7 @@ name: line from the distal-most end of the longest phalax of the foot to the pos
 def: "An anatomical line of a hindfoot that connects the end of the longest digit to the posterior-most part of a foot." [https://orcid.org/0000-0003-2699-3066]
 intersection_of: UBERON:0006800 ! anatomical line
 intersection_of: connects UBERON:7500039 ! posterior of pes
-intersection_of: connects UBERON:7500041 ! tip of digit
+intersection_of: connects UBERON:7500041 ! obsolete tip of digit
```

Comment:
why isn't this flagged automatically?

Reply:
I fixed this!

Reply:
The check was on master but not yet on this branch - now it is, so will fail. @nicolevasilevsky this is indeed _not_ fixed at the moment!

Reply:
Oh it was fixed..